src/components/promotional-card.tsx

```diff
@@ -0,0 +1,71 @@
+"use client";
+
+import Image from "next/image";
+
+import { CardType } from "@/lib/types";
+
+interface Cardprops {
+  type: CardType;
+}
+
+const getColor = (type: CardType) => {
+  return type === CardType.Collaborate ? "blue" : "primary";
+};
+
+const info: Record<CardType, { name: string; text: string; image: string }> = {
+  [CardType.Collaborate]: {
+    name: "colaborador",
+    text: "Podes juntar-te a estes departamentos e contribuir para fazer o teu núcleo funcionar.",
+    image: "collaborator.svg",
+  },
+  [CardType.Membership]: {
+    name: "sócio",
+    text: "Participa em todos os nossos eventos gratuitamente, e usufrui de muitos mais benefícios.",
+    image: "partnercard.svg",
+  },
+};
+
+const PromotionalCard = ({ type }: Cardprops) => {
+  const color = getColor(type);
+
+  return (
+    <div
+      className={`relative flex min-h-60 w-full flex-col items-center justify-between gap-4 overflow-hidden rounded-2xl p-6 text-white min-[950px]:min-h-0 min-[950px]:flex-row min-[950px]:p-8 bg-${color}`}
+    >
+      {/* Card image */}
+      <div
+        className={`absolute bottom-0 hidden min-[340px]:block ${type == CardType.Collaborate ? "left-12 h-24 min-[950px]:h-28" : "left-10 h-20 min-[950px]:h-24"} min-[950px]:bottom-0`}
```

Comment:
There are some overlaps on the mobile view, maybe use a grid instead of absolute to fix this. This happens at least on the iPhone 11 Pro.

Reply:
I think that only happens because the padding in the test `page.tsx` is bigger than what it will actually be on mobile (it's currently set to `p-8` and it's `p-5` on mobile). This is how iPhone 11 Pro looks like to me with `p-5`:

![image](https://github.com/user-attachments/assets/3f2de0a8-2186-4108-a672-0076acb4e3d7)

Can you try to recreate?